ts/Series/Bullet/BulletSeries.ts

```diff
@@ -135,7 +135,14 @@ class BulletSeries extends ColumnSeries {
              *
              * @since   6.0.0
              */
-            borderWidth: 0
+            borderWidth: 0,
+
+            /**
+             * The border radius of the rectangle representing the target.
+             *
+             * In styled mode, use class `highcharts-bullet-target` instead.
```

Comment:
We can't set the border radius of a shape in CSS, so we should remove the reference to styled mode.